x-pack/plugins/alerting/server/saved_objects/index.ts

```diff
@@ -176,6 +176,22 @@ export function setupSavedObjects(
         createdAt: {
           type: 'date',
         },
+        end: {
```

Comment:
We should add a `modelVersions` entry with a `mappings_addition` entry in it's changes ([recently added tutorial](https://docs.elastic.dev/kibana-dev-docs/tutorials/saved-objects#migrations))

Reply:
Thanks for reviewing! Added in https://github.com/elastic/kibana/pull/179975/commits/a15d76b7a7c9812e2b2e1e90c45a7efb0b674da7

Reply:
Nice cleanup! Although it looks like the `changes` array is still empty. We should add `end: {...}` there (just the diff of the new mappings).